tests/driver.sh

```diff
@@ -296,4 +296,17 @@ int main()
 }
 EOF
 
+# break from while
```

Comment:
The existing section `loop` already contains `while(1) break; exit(10);`. You should enable it along with the similar cases.